typescript/sdk/src/consts/environments/mainnet.json

```diff
@@ -194,5 +194,20 @@
     "aggregationHook": "0x8464aF853363B8d6844070F68b0AB34Cb6523d0F",
     "protocolFee": "0xd83A4F747fE80Ed98839e05079B1B7Fe037b1638",
     "validatorAnnounce": "0x2fa5F5C96419C222cDbCeC797D696e6cE428A7A9"
+  },
+  "viction": {
+    "merkleRootMultisigIsmFactory": "0x2C1FAbEcd7bFBdEBF27CcdB67baADB38b6Df90fC",
+    "messageIdMultisigIsmFactory": "0x8b83fefd896fAa52057798f6426E9f0B080FCCcE",
+    "aggregationIsmFactory": "0x8F7454AC98228f3504Bb91eA3D8Adafe6406110A",
+    "aggregationHookFactory": "0xEb9FcFDC9EfDC17c1EC5E1dc085B98485da213D6",
+    "routingIsmFactory": "0x1052eF3419f26Bec74Ed7CEf4a4FA6812Bc09908",
+    "mailbox": "0x2f2aFaE1139Ce54feFC03593FeE8AB2aDF4a85A7",
+    "proxyAdmin": "0x0761b0827849abbf7b0cC09CE14e1C93D87f5004",
+    "validatorAnnounce": "0x2fa5F5C96419C222cDbCeC797D696e6cE428A7A9",
+    "interchainSecurityModule": "0xBD70Ea9D599a0FC8158B026797177773C3445730",
+    "merkleTreeHook": "0x149db7afD694722747035d5AEC7007ccb6F8f112",
+    "storageGasOracle": "0x19dc38aeae620380430C200a6E990D5Af5480117",
+    "interchainGasPaymaster": "0x0D63128D887159d63De29497dfa45AFc7C699AE4",
+    "protocolFee": "0xd83A4F747fE80Ed98839e05079B1B7Fe037b1638"
```

Comment:
Just a reminder to add the testRecipient addresses here too before merging please :)